src/assets/locales/fr.json

```diff
@@ -90,25 +117,20 @@
   "close-room-modal.message": "En fermant cette salle, vous allez exclure tous les participants, vous y compris, et la fermer définitivement.{linebreak}Confirmer ? Cette action ne peut pas être annulée.",
   "close-room-modal.room-name-match-error": "Le nom de la salle ne correspond pas",
   "close-room-modal.title": "Fermer la salle",
-  "close-room-modal.type-to-confirm": "Entrer le nom de la salle pour confirmer: {roomName}",
+  "close-room-modal.type-to-confirm": "Entrer le nom de la salle pour confirmer : {roomName}",
   "content-menu.objects-menu-button": "Objets",
   "content-menu.people-menu-button": "Personnes ({presenceCount})",
```

Comment:
@vincentfretin what if we switch {presenceCount} and "personnes" ?

Reply:
If you want "Personne", "1 personne", "2 personnes", this would be better but require a change in the code.
For me with the parentheses here, it's just a counter. Like for "Objets (2)" next to it.

Reply:
Oh you are right

Reply:
Oh you are right